CMakeLists.txt

```
@@ -221,6 +222,10 @@ if(USE_CUDA)
   add_subdirectory(${PROJECT_SOURCE_DIR}/gputreeshap)
 
   find_package(CUDAToolkit REQUIRED)
+
+  if(BUILD_WITH_EXTERNAL_CCCL)
+    find_package(CCCL REQUIRED CONFIG COMPONENTS cub)
```

Do we need Thrust here too?

```suggestion
    find_package(CCCL REQUIRED CONFIG COMPONENTS cub thrust)
```